### ISSUE TITLE
Define preset value for title attribute

### ISSUE THREAD

**Issue by heikobornholdt** [infopark/reactor#18]
I want to define a preset value for the title attribute of a class.

**Comment by tomaszp-infopark** [comment]
At the moment there is no official API for setting presets for built-in attributes.

I will put this request on the feature list for the next release (coming next week).

As a temporary workaround you could use the following:
```ruby
class TitlePreset < Reactor::Migration
  def self.up
    update_class name: 'publication' do
      @preset_attrs[:title]  = 'Title preset'
    end
  end

  def self.down
    false
  end
end
```

Next release will include proper API for this:

```ruby
  preset :title, 'Title preset'
```

**Comment by heikobornholdt** [comment]
Thank you. The workaround work.

**Comment by tomaszp-infopark** [comment]
I've just released `1.12.0.beta1` which contains this feature. Would you mind taking a look?

**Comment by heikobornholdt** [comment]
`preset :title` is working well here :+1: .
